packages/terra-application-links/src/tabs/_TabMenu.jsx

```diff
@@ -11,6 +12,10 @@ const propTypes = {
    * Child tabs to be placed in the tab menu.
    */
   children: PropTypes.array,
+  /**
+   * intl object programmatically imported through injectIntl from react-intl.
+   * */
+  intl: intlShape.isRequired,
```

Comment:
Please use the `@private`syntax wherever applicable. Reference - https://github.com/cerner/terra-core/pull/2507/files#r303638053

Reply:
Done, changed as part of : d5474e9